### ISSUE TITLE
Finetuning VoCov2 on Classification task in UNet manner

### ISSUE THREAD

**Issue by dongdongtong** [Luffy03/Large-Scale-Medical#28]
Hi, congratulations on the new impressive work.

Here I got one question about the finetuning model architecture of VoCo on the classification task: CC-CCII. I notice here, the arch of the classification, is actually formulated in a UNet manner, that is, encoder-decoder. You use the upsampled feature maps for the classification task, not the bottleneck feature maps from the pretrained VoCo. So is there any insights on a UNet for finetuning VoCo on classification?

https://github.com/Luffy03/Large-Scale-Medical/blob/ccae1410f97d2c1a196abb4ae7ee96a482215ea8/Downstream/monai/CC-CCII/model.py#L136-L156

**Comment by Luffy03** [comment]
Hi, thank you very much for your attention to our work!
We adopt this setting for two primary reasons: (1) First, it allows us to use more pre-trained parameters for downstream classification (2) Second, with a decoder for upsampling, the model can recognize more refined lesions. Finally, we find that this setting can achieve better performance.

**Comment by dongdongtong** [comment]
Thanks for your insightful answer, this is very interesting. 
And I am curious that the performance gap if we use the bottleneck feature maps compared to the upsampled feature maps, empirically in your conducted experiments?

**Comment by Luffy03** [comment]
Sorry that we did not save the results of this setting. I remember that this setting would drop about 1-2% accuracy on the CCII dataset, but I am not sure. Maybe you need to conduct the experiments to see the details.

**Comment by dongdongtong** [comment]
Many thanks on your in-time and helpful answer. This inspires me a lot. Thanks again!~

**Comment by Luffy03** [comment]
You are welcome. Feel free to raise issues if you have any other problems.
